EIPS/eip-5593.md

```diff
@@ -71,23 +71,16 @@ Test suite link needs to be created and linked here still.
 
 ## Security Considerations
 
-### User Enables Developer Mode 
+### User Enables Developer Mode
 
-Oftentimes developers require the ability to develop dApps locally in order to test their website and develop while hosting their dApp on http://localhost. In this case localhost would be blocked and compatibility issues would arise when developing a dApp locally. In order to increase compatibility for dApp developers a toggle to disable the check for the localhost can be considered. If this were to be extended beyond the localhost origin it could be used as a means to convince users to enable developer mode in order to subvert the guards put in place by this EIP. Therefore, implementations should be cautious when extending this developer toggle beyond the scope of the localhost origin.
+Oftentimes developers require the ability to develop dApps locally in order to test their website and develop while hosting their dApp on `http://localhost`. In this case localhost would be blocked and compatibility issues would arise when developing a dApp locally. In order to increase compatibility for dApp developers a toggle to disable the check for the localhost can be considered. If this were to be extended beyond the localhost origin it could be used as a means to convince users to enable developer mode in order to subvert the guards put in place by this EIP. Therefore, implementations should be cautious when extending this developer toggle beyond the scope of the localhost origin.
 
-## Privacy Considerations
+### Privacy Considerations
```

Comment:
AFAICT in EIP-1 it's acceptable to add additional categories and this is up to the author to determine. Why is the bot enforcing this specific template when it's not followed by all EIPs?